src/Router.php

```diff
@@ -104,14 +104,29 @@ public function setOptions(array $options)
         }
 
         if (!empty($options['routes'])) {
-            foreach ($options['routes'] as $path => $routeConfig) {
-                $defaults = isset($routeConfig['defaults']) ? $routeConfig['defaults'] : [];
-                $requirements = isset($routeConfig['requirements']) ? $routeConfig['requirements'] : [];
-                $this->addRoute(new Route($path, $defaults, $requirements));
+            foreach ($options['routes'] as $name => $routeConfig) {
+                if (array_key_exists('path', $routeConfig)) {
+                    $path = $routeConfig['path'];
+                } else {
+                    $path = $name;
+                }
```

Comment:
I think this if-logic would be better placed in the `addRouteFromConfig`-method. You could also rename the `$name` variable and the `$path` parameter to `$nameOrPath` to make the intent more clear.